tests/js/server/aql/aql-graph.js

```diff
@@ -1646,6 +1672,68 @@ function kPathsTestSuite() {
   };
 }
 
+function allShortestPathsTestSuite() {
+  const gn = "UnitTestGraph";
+  const vn = "UnitTestV";
+  const en = "UnitTestE";
+
+  return {
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief set up
+////////////////////////////////////////////////////////////////////////////////
+
+    setUpAll: function () {
+      gm._create(gn, [gm._relation(en, vn, vn)]);
+
+      ["s", "t", "a", "b", "c", "d", "e", "f", "g"].map((elem) => {
+        db[vn].insert({_key: elem});
+      });
+
+      [
+        ["s", "a"], ["s", "b"], ["a", "b"], ["a", "c"], ["b", "c"],
+        ["c", "d"], ["d", "t"],
+        ["c", "e"], ["e", "t"],
+        ["c", "f"], ["f", "t"],
+        ["c", "g"], ["g", "t"]
+      ].map(([a, b]) => {
+        db[en].insert({_from: `${vn}/${a}`, _to: `${vn}/${b}`});
+      });
+    },
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief tear down
+////////////////////////////////////////////////////////////////////////////////
+
+    tearDownAll: function () {
+      gm._drop(gn, true);
+    },
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief checks if we are able to find all shortest paths when using ANY direction with ALL_SHORTEST_PATHS
+/// One of the edges is used in both directions.
+////////////////////////////////////////////////////////////////////////////////
+
+    testAllShortestPathsAnyUseEdgeTwice: function () {
+      // TODO Anthony: Clarify what "One of the edges is used in both directions" means in ASP context
```

Comment:
Is this TODO still open?
i am missing a bit context here, can we elaborate a bit more on this question? (Happy to do so on slack)

Reply:
ah, it's out of date, thank you for noticing

addressed in 7dd39c35d94